doc/community-build/cloud/AWS/README.md

```diff
@@ -77,19 +77,19 @@ This file consists of the procedure to compile complete CORTX stack and Deploy o
 
 ### 1. CORTX Build
 
-   We will use [cortx-build](https://github.com/Seagate/cortx/pkgs/container/cortx-build) docker image to compile entire CORTX stack. Please follow [CORTX compilation](https://github.com/Seagate/cortx/blob/main/doc/community-build/Generate-Cortx-Build-Stack.md) steps for compilation.
+   We will use [cortx-build](https://github.com/Seagate/cortx/pkgs/container/cortx-build) docker image to compile entire CORTX stack. Please follow [CORTX compilation](https://github.com/Seagate/cortx/blob/main/doc/community-build/archive/Generate-Cortx-Build-Stack.md) steps for compilation.
```

Comment:
CORTX Compilation should link to https://github.com/Seagate/cortx/blob/main/doc/community-build/docker/cortx-all/README.md

Reply:
Done